haystack/tracing/datadog.py

```diff
@@ -60,12 +64,28 @@ def __init__(self, tracer: "ddTracer") -> None:
         ddtrace_import.check()
         self._tracer = tracer
 
+    @staticmethod
+    def _get_span_resource_name(operation_name: str, tags: Optional[Dict[str, Any]]) -> Optional[str]:
+        """
+        Get the resource name for the Datadog span.
+        """
+        if operation_name == _COMPONENT_RUN_OPERATION_NAME and tags:
+            component_type = tags.get(_COMPONENT_TYPE_KEY, "")
+            component_name = tags.get(_COMPONENT_NAME_KEY, "")
+            resource_name = f"{component_type} {component_name}".strip()
```

Comment:
Could we make a slight change here based on the suggestion from the issue?
```suggestion
            resource_name = f"{component_type}: {component_name}".strip()
```